deploy-board/deploy_board/webapp/account_utils.py

```diff
@@ -0,0 +1,54 @@
+from deploy_board.settings import AWS_PRIMARY_ACCOUNT, AWS_SUB_ACCOUNT
+from .helpers import clusters_helper, accounts_helper, deploys_helper
+from . import agent_report
+
+def create_legacy_ui_account(account_id):
```

Comment:
`util` naming is an anti pattern. Let's use more meaningful names because `util` may means anything.
Maybe rename file to just `accounts.py`?